### ISSUE TITLE
Spreadsheet brim does not match

### ISSUE THREAD

**Issue by kirk223** [bambulab/BambuStudio#5390]
### Bambu Studio Version

1.10.1.50

### Where is the application from?

Bambu Lab Official website

### OS version

macOS 13.7.1

### Printer

BambuLab P1S

### How to reproduce

Brim type  does not work properly
Mark brim type but the actual brim type is wrong. For example:
Select outer brim - nothing happens when slicing
check the spreadsheet and the support info shows painted.
Select inner brim
the spreadsheet shows outer brim 
outer brim is produced when slicing
![Brim does not match properly](https://github.com/user-attachments/assets/46885d19-08b8-4dfd-8117-1d018393be1f)

### Actual results

brim types are not matching properly and slices the wrong type

### Expected results

Selected brim should match up properly.  Happens on all files. The problem appeared when the painted brim was added.

### Project file & Debug log uploads

[Mrs. Claus_Torso_MC Kirk copy.3mf.zip](https://github.com/user-attachments/files/17880261/Mrs.Claus_Torso_MC.Kirk.copy.3mf.zip)

### Checklist of files to include

- [X] Log file
- [X] Project file

**Comment by MackBambu** [comment]
confirm

**Comment by kirk223** [comment]
Confirm

Kirk

**Comment by nnobuddy** [comment]
Confirmed.

**Comment by kirk223** [comment]
Im sorry but are you looking for additional information from me?

Kirk

**Comment by nnobuddy** [comment]
> Im sorry but are you looking for additional information from me? Kirk
> […](#)
> On Nov 27, 2024 at 11:04 AM -0500, nnobuddy ***@***.***>, wrote: Confirmed. — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored the thread.Message ID: ***@***.***>

No, I just confirmed that your original report is true and that there is an issue in Bambu Studio.
Thx very much for reporting.

**Comment by MackBambu** [comment]
fixed
try: https://github.com/bambulab/BambuStudio/releases/tag/v01.10.02.64
